### PR TITLE
Attaching PMSP LiDAR 3D Yaml Dataset file

### PR DESCRIPTION
**Pull request by feromes**
Hi,

I am attaching São Paulo 3D LiDAR YAML in order to update datasets. The service is already available, up and running.

Thanks for all of you, AWS OpenData Team,

Regards,

Fernando Gomes

### CODE REVIEW

File: datasets/pmsp-lidar.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+Name: PMSP LiDAR Point Cloud
```
Comment (jflasher) [inline datasets/pmsp-lidar.yaml:1]: Please define PMSP in the title here.

File: datasets/pmsp-lidar.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+Description: The objective of the M3DC of the São Paulo City Hall is to publish LiDAR point cloud data. The initial data was acquired in 2017 by aerial surveying and future data will be added. This public accessible data set is provided in the format [Entwine Point Tiles](https://entwine.io/entwine-point-tile.html) as a lossless octree, full density, based on [LASzip] ( https://laszip.org) (LAZ) encoding.
```
Comment (jflasher) [inline datasets/pmsp-lidar.yaml:2]: Please define M3DC the first time.

File: datasets/pmsp-lidar.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+Documentation: https://github.com/geoinfo-smdu/M3DC
+Contact: geosampa@prefeitura.sp.gov.br
+ManagedBy: "[GeoSampa - o mapa digital da cidade de São Paulo](http://geosampa.prefeitura.sp.gov.br)"
+UpdateFrequency: Local survey executed by demand generates new data as local point clouds.
+Tags:
```
Comment (jflasher) [inline datasets/pmsp-lidar.yaml:7]: Please only use tags from https://github.com/awslabs/open-data-registry/blob/master/tags.yaml. Also, please add `aws-pds` as a tag.

File: datasets/pmsp-lidar.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  - urban/city model
+  - lidar point cloud
+  - digital elevation/surface/terrain model
+  - geospatial
+  - building
+  - cartography
+  - forestry/vegetation
+  - urban forest
+  - land use
+License: GNU General Public License v3.0 https://www.gnu.org/licenses/gpl-3.0.html
```
Comment (jflasher) [inline datasets/pmsp-lidar.yaml:18]: Please change this to markdown like "[GNU General Public License v3.0](https://www.gnu.org/licenses/gpl-3.0.html)"

File: datasets/pmsp-lidar.yaml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+Resources:
+  - Description: São Paulo city's 3D LiDAR - EPT(Entwine Point of Clouds).
```
Comment (jflasher) [inline datasets/pmsp-lidar.yaml:20]: I think you can drop "of" in EPT.

**Comment by feromes** [comment]
Hi @jflasher 

I have just commit changes in order to fix the issues you  pointed. #

Thanks,

File: datasets/pmsp-lidar.yaml [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+    - Title: Fusion
+      URL: http://forsys.sefs.uw.edu/fusion/fusionlatest.html
+      AuthorName: US Department of Agriculture - Forest Service
+      AuthorURL: https://www.usda.gov/
+  Publications:
+    - Title: Describing the Vertical Structure of Informal Settlements on the Basis of LiDAR Data – A Case Study for Favelas (Slums) in Sao Paulo City
+      URL: https://doi.org/10.5194/isprs-annals-IV-2-W5-437-2019
+      AuthorName: S. C. L. Ribeiro, M. Jarzabek-Rychard, J. P. Cintra, H.-G. Maas
+    - Title: Primeiro ciclo do ouro em São Paulo – conhecendo sua importância com o auxílio da Geoarqueologia
+      AuthorName: F. A. N. da Silva
```
Comment (jflasher) [inline datasets/pmsp-lidar.yaml:49]: @feromes is there a URL for this publication? Please either add or delete this entry. I think this should be good to go after that.

**Comment by jflasher** [comment]
Looks good. I am going to bring over to my own branch to make a few text changes and then will merge.

**Comment by jflasher** [comment]
Merged via https://github.com/awslabs/open-data-registry/pull/538

**Pull request closed without merging**